crates/block-producer/src/store/mod.rs

```diff
@@ -230,7 +187,7 @@ impl Store for DefaultStore {
         Ok(tx_inputs)
     }
 
-    async fn get_block_inputs(
+    pub async fn get_block_inputs(
```

Comment:
Should we add `instrument` to this method as well? (not sure why it wasn't instrumented before).

Reply:
I was thinking we'd do a pass over all instrumentation as part of the monitoring work. But I may as well instrument this while I'm here :) Thanks.

Reply:
Tackled in #574